lua/completion/source.lua

```diff
@@ -14,15 +14,15 @@ local M = {}
 local complete_items_map = {
   ['lsp'] = {
     trigger = lsp.triggerFunction,
-    callback = lsp.getCallback,
+    handler = lsp.getHandler,
     item = lsp.getCompletionItems
   },
   ['snippet'] = {
     item = snippet.getCompletionItems
   },
   ['path'] = {
     item = path.getCompletionItems,
-    callback = path.getCallback,
+    handler = path.getHandler,
```

Comment:
what does this do?

It's not necessarily tru we have to change the name of "callback" -> "handler" for every scenario.

Reply:
That registers a default callback for the builtin `path` completion source. As you said, we don't _have_ to change what it's called, although I didn't want to mistakenly direct anyone browsing the code to an older functionality.